managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -3622,4 +3622,35 @@ public boolean isCacheReadEntry() {
     public ManagedLedgerConfig getConfig() {
         return config;
     }
+
+    /***
+     * Create a non-durable cursor and copy the ack stats.
+     */
+    public CompletableFuture<ManagedCursor> duplicateToNonDurableCursor(String nonDurableCursorName) {
```

Comment:
Maybe we don't need to use CompletableFuture as a result, there is no async method.